src/com/cft/contactmerge/ContactImporter.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * ContactImporter is responsible for calling the right importer class
+ * (xml, tsv, csv, etc), extract contact information from the specified file
+ * and return the extract list of contacts when requested.
+ */
+package com.cft.contactmerge;
+import java.util.Collection;
+
+public class ContactImporter {
+    public ContactImporter(SupportedFileType fileType, String fileName, Collection<ColumnMap> columnMap )
+    {
+        // TODO: Implementation needed
+    }
+
+    public Collection<Contact> getContacts()
+    {
+        // TODO: Implementation needed
+        return null;
+    }
+}
```

Comment:
This is just a thought... but we may want to change this to FileImporter and do one of two things...
Use polymorphism for getContacts() when handling different file types.
-or-
Use method overloading, and passing the filetype as an argument.

Reply:
I prefer "Contact Importer" because it clearly identifies the objective. We aren't trying to load a file... We are loading contacts from a file. The result is a list if contacts. My thought is that the polymorphism will happen inside getContacts as there will be a class for each type of file.